openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -739,6 +743,23 @@ def validate_file_volumes_count(self, h_node, h_server, node_ip):
             "Failed to verify volume count Expected:'{}', Actual:'{}'".format(
                 len(h_volume_list['volumes']), len(vol_list)))
 
+    def match_volume_by_prefix(self, prefix):
+        """Match PVC, PV, Heketi and Gluster volume
+
+        Args:
+            prefix (str): Start of the unique string.
+        """
+        match_pvc_and_pv(self.h_node, prefix)
+        h_vols = heketi_volume_list_by_name_prefix(
+            self.h_node, self.h_server, prefix, json=True)
```

Comment:
I think h_node and h_server are not defined anywhere in baseclass. Better to add them in function parameters.

Reply:
+1

Reply:
Done